recipes/kaolin/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "kaolin" %}
+{% set version = "0.9.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIAGameWorks/kaolin/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7fa6b794d83fca9b4a1eb93b13ead60767b42d5a8a25162ee73be3cead1a8c8c
+
+build:
+  number: 0
+  skip: true  # [cuda_compiler_version == "None" or cuda_compiler_version == "10.2"]
+
+requirements:
+  build:
+    - cython
+    - cuda
+    - {{ compiler('cxx') }}
+    - python
+  host:
+    - pip
+    - python 
```

Would add `numpy` here since it appears to be used by [the Cython code]( https://github.com/NVIDIAGameWorks/kaolin/blob/v0.9.1/kaolin/cython/ops/mesh/triangle_hash.pyx#L26 )

```suggestion
    - python
    - numpy
```